components/search/DateFilter.stories.js

```diff
@@ -0,0 +1,38 @@
+import { storiesOf } from '@storybook/vue';
+import DateFilter from './DateFilter.vue';
+
+const i18n = {
+  locale: 'en',
+  messages: {
+    en: {
+      dateFilter: {
+        legend: 'Date range',
+        startDate: 'Start date',
+        endDate: 'End date'
+      }
+    }
+  }
+};
+
+storiesOf('Search', module)
```

Comment:
Could you add an action method to this so that it observes the event emitted and logs to the Storybook UI the arguments sent to the event handler?